rs/p2p/quic_transport/README.adoc

```diff
@@ -8,33 +8,30 @@ The IC Transport layer enables message delivery between nodes within a subnet.
 
 == Requirements ==
 
-[cols="1,1"]
+[cols="3,3,3"]
 |===
-| **What ?**
-| **Why ?**
+| **Capability**
+| **Purpose**
+| **Implementation Requirement**
 
+| RPC-like library
+| Simplifies application development by abstracting communication between nodes, allowing developers to focus on application logic instead of low-level networking.
+| Ensure https://en.wikipedia.org/wiki/Reliability_(computer_networking)[reliable data delivery] so the sender can confirm successful message delivery to the recipient.
 
-| https://en.wikipedia.org/wiki/Reliability_(computer_networking)[Reliable data delivery] - the sender knows if a message was delivered succesfully to the recepient.
-| Applications on top of the transport layer are simpler to implement when this abstraction exists. E.g. they don't need to re-implement TCP or QUIC.
+| Multitenancy of applications
```

Comment:
```suggestion
| Multitenancy of applications.
```

Reply:
I would prefer to keep it without the dot because it is more of a title than a full sentence